MuonAnalysis/MuonAssociators/interface/PropagateToMuon.h

```diff
@@ -4,12 +4,8 @@
 //
 
 /**
-  \class    PropagateToMuon PropagateToMuon.h "MuonAnalysis/MuonAssociators/interface/PropagateToMuon.h"
-  \brief    Propagate an object (usually a track) to the second muon station.
-            Support for other muon stations will be added on request.
-            
-  \author   Giovanni Petrucciani
-  \version  $Id: PropagateToMuon.h,v 1.3 2010/07/01 07:41:09 gpetrucc Exp $
+  \class    PropagateToMuon PropagateToMuon.h
+  "MuonAnalysis/MuonAssociators/interface/PropagateToMuon.h" \brief Propagate an object (usually a track) to the second (default) or first muon station.
```

Comment:
Here I meant that `"MuonAnalysis/MuonAssociators/interface/PropagateToMuon.h"` should be moved to the line above (see format of the previous impl).

Reply:
Done.